extensions/xcode/src/open-url-in-simulator.command.tsx

```diff
@@ -0,0 +1,37 @@
+import { XcodeSimulatorService } from "./services/xcode-simulator.service";
+import { Clipboard, closeMainWindow, showToast, Toast } from "@raycast/api";
+
+interface OpenSimulatorArguments {
+  url?: string;
+}
+
+export default async (props: { arguments: OpenSimulatorArguments }) => {
+  const toast = await showToast({
+    style: Toast.Style.Animated,
+    title: "Opening URL in Simulator...",
+  });
+  try {
+    const url = props.arguments.url ?? (await Clipboard.readText());
+    if (!url) throw Error("Please provide or copy a valid URL.");
```

Comment:
Instead of Try/Catch it would be good to split up the error paths and show a different error e.g. in this case where no URL is available.

Reply:
Why splitting instead of just throwing an `Error` with a different message?